### DISCUSSION TITLE
How to activate RTL support

### DISCUSSION THREAD

**Discussion by Xoshbin** [filamentphp/filament#449] [Help] [answered]
Hi, I see in the releases that fillament supports RTL layout, but I can't find how to activate it please? Thank you

**Comment by danharrin** [comment, marked as answer]
Hey, you need to switch to an app locale that supports it. For example, [Arabic](https://github.com/laravel-filament/filament/blob/main/resources/lang/ar/layout.php).

**Reply by Xoshbin** [reply]
Thank you, I totally forgot about that.
